packages/cms/src/schemas/schema.ts

```diff
@@ -6,47 +6,9 @@ import createSchema from 'part:@sanity/base/schema-creator';
  * Import the ones using named exports
  */
 import * as allDocuments from './documents';
-// documents are items that are published/queriable
-// some of these are 'singletons' but that's not enforced by the API
-// import siteSettings from "./documents/siteSettings";
-import article from './documents/article';
-import editorial from './documents/editorial';
-import behaviorPage from './documents/pages/behavior-page';
-import cijferVerantwoording from './documents/pages/cijfer-verantwoording-page';
-import deceasedPage from './documents/pages/deceased-page';
-import escalationLevelPage from './documents/pages/escalation-level-page';
-import hospitalPage from './documents/pages/hospital-page';
-import intensiveCarePage from './documents/pages/intensive-care-page';
-import overDitDashboard from './documents/pages/over-dit-dashboard-page';
-import overRisicoNiveaus from './documents/pages/over-risico-niveaus-page';
-import positiveTestsPage from './documents/pages/positive-tests-page';
-import reproductionPage from './documents/pages/reproduction-page';
-import sewerPage from './documents/pages/sewer-page';
-import topicalPage from './documents/pages/topical-page';
-import vaccinationsPage from './documents/pages/vaccinations-page';
-import veelgesteldeVragenGroepen from './documents/pages/veelgestelde-vragen-groepen-page';
-import veelgesteldeVragen from './documents/pages/veelgestelde-vragen-page';
-import toegankelijkheid from './documents/toegankelijkheid';
-import localeBlock from './locale/locale-block';
-import localeRichContentBlock from './locale/locale-rich-content-block';
-// These 2 locale helpers are technically objects too, but we keep them grouped here
-// so it's easier to scan over the different imports and recognize patterns
-import localeString from './locale/locale-string';
-import localeText from './locale/locale-text';
-//objects are building blocks, but not queryable in itself
-import collapsible from './objects/collapsible';
-import decoratedLink from './objects/decorated-link';
-import faqQuestion from './objects/faq-question';
-import lineChart from './objects/line-chart';
-import milestone from './objects/milestone';
-import titleDescriptionBlock from './objects/title-description-block';
-import lockdown from './restrictions/lockdown';
-import restriction from './restrictions/restriction';
-import restrictionCategory from './restrictions/restriction-category';
-// routekaart en maatregelen
-import restrictionGroup from './restrictions/restriction-group';
-import restrictionGroupLockdown from './restrictions/restriction-group-lockdown';
-import roadmap from './restrictions/roadmap';
+import * as localeHelpers from './locale';
+import * as allObjects from './objects';
+import * as restrictionDocuments from './restrictions';
```

Comment:
nit; I think the names of the folders are sufficient, as the concat below doesn't care. It prevents you from coming up with new names for things. So you could write 

```suggestion
import * as locale from './locale';
import * as objects from './objects';
import * as restrictions from './restrictions';
```
Then also rename allDocuments to documents I guess. Currently it's a little odd that there's both allDocuments and restrictionDocuments...